tools/webhook_test.py

```diff
@@ -126,6 +126,8 @@ def set_init(webhook_type):
             "message": {
                 "gym_id": 0,
                 "gym_name": "unknown",
+                "park": None,
+                "sponsor": 4,
```

Comment:
should probably default this to none

Reply:
I might remove based on above comments....